features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -855,6 +884,96 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }
     };
 
+    const hybridFlowResponseTypeMap: Map<string, string> = new Map([
+        [
+            t("applications:forms.inboundOIDC.sections.hybridFlow.hybridFlowResponseType." +
+            "fields.children.code_token.label"),
+            ApplicationManagementConstants.CODE_TOKEN
+        ],
+        [
+            t("applications:forms.inboundOIDC.sections.hybridFlow.hybridFlowResponseType.fields." +
+            "children.code_idtoken.label"),
+            ApplicationManagementConstants.CODE_IDTOKEN
+        ],
+        [
+            t("applications:forms.inboundOIDC.sections.hybridFlow.hybridFlowResponseType.fields." +
+            "children.code_idtoken_token.label"),
+            ApplicationManagementConstants.CODE_IDTOKEN_TOKEN
+        ]
+    ]);
+
+    const getHybridFlowResponseTypesHintDescription = (value: string): string => {
+        switch (value) {
+            case ApplicationManagementConstants.CODE_TOKEN:
+                return t("applications:forms.inboundOIDC.fields.hybridFlow.hybridFlowResponseType.children." +
+                    "code_token.hint");
+            case ApplicationManagementConstants.CODE_IDTOKEN_TOKEN:
+                return t("applications:forms.inboundOIDC.fields.hybridFlow.hybridFlowResponseType.children." +
+                    "code_idtoken_token.hint");
+            default:
+                return null;
+        }
+    };
+
+    const getHybridFlowResponseTypes = () => {
```

Comment:
Add the return type and a TS doc comment. Check the other new functions as well.

Reply:
Created an issue to bring in automated tsdoc rule enforcing.
https://github.com/wso2/product-is/issues/20583

Reply:
fixed with : https://github.com/wso2/identity-apps/pull/6271/commits/393ec2c21d1d125976e3fc24227de82146de28ee